critique.py

```diff
@@ -153,13 +256,14 @@ def walk_function(registrar, path, files):
     if path.find('.git') > -1:      # skip the Git VCS directory
         return
     for nm in files:
-        registrar.add(path, Critic(path, nm))
+        if os.path.splitext(nm)[1] not in ['.txt','.py','.rst','.md','.in'] and nm[0] != '.': # skip other types of file
```

Comment:
if os.path.splitext(nm)[1] not in extensions and nm[0] != '.': # skip other types of file